### DISCUSSION TITLE
How to get automatic evaluation details

### DISCUSSION THREAD

**Discussion by PierreCailliot** [Estia-advanced-programming/pandora-public#87] [Q&A] [answered]
# What is you question about : 
![image](https://user-images.githubusercontent.com/80093199/110439196-737eea80-80b7-11eb-9b46-e1ec46f19d6b.png)
avez vous les details de la note attribué pour chaque milestone?

**Comment by dhmmasson** [comment]
Yes

**Comment by dhmmasson** [comment, marked as answer]
You can go to the checks on top of the pull requests 

Either click on the automatic milestone in and then look into run tests > features to see which feature pass
![Screen Shot 2021-03-09 at 09 39 12](https://user-images.githubusercontent.com/5263585/110445678-a7a9d980-80be-11eb-9479-3dbd28acd3eb.png)

Or Download the report in the artifacts
